docs/content/doc/packages/cran.en-us.md

```diff
@@ -0,0 +1,96 @@
+---
+date: "2023-01-01T00:00:00+00:00"
+title: "CRAN Packages Repository"
+slug: "packages/cran"
+draft: false
+toc: false
+menu:
+  sidebar:
+    parent: "packages"
+    name: "CRAN"
+    weight: 35
+    identifier: "cran"
+---
+
+# CRAN Packages Repository
+
+Publish [CRAN](https://cran.r-project.org/) packages for your user or organization.
```

Comment:
```suggestion
Provide [CRAN-like](https://cran.r-project.org/) packages for your user or organization.
```

CRAN is a group/institution on it's own. When providing packages via custom repositories, the R community usually names them "CRAN-like" packages as they only need to follow a specific directory structure to work with `install.packages()`.

CRAN is quite picky on publishing packages, so I don't think they would like seeing a sentence indicating that one could publish "CRAN packages" on their own :)

Reply:
There was a similar discussion for Cargo in https://github.com/go-gitea/gitea/pull/21888#discussion_r1033203130 (the question is gone (?), only my answer is there).

Software using "CRAN":
https://www.jfrog.com/confluence/display/JFROG/CRAN+Repositories
https://gitlab.com/gitlab-org/gitlab/-/issues/36892
https://cloudsmith.com/cran-repository/
only Nexus uses "R":
https://help.sonatype.com/repomanager3/nexus-repository-administration/formats/r-repositories

I don't have R knowledge, so I don't have a favorite.

Reply:
GitLab [also calls it CRAN](https://gitlab.com/gitlab-org/gitlab/-/issues/36892), so I think this point is naught.

Reply:
There is really no such thing as "CRAN packages" despite the ones which are "officially" released. The common name is "R package" which are published in a CRAN-like repository/registry. The "-like" is just because it follows a fixed directory scheme so that `install.packages` works. 

Everybody in the community says "R package". These can be "published on CRAN" but are an "R package" per se, no matter if hosted on GitHub or published on CRAN.

I think that calling the registry/repository "CRAN" might be ok but then we should change the wording above to "Publish R packages to a CRAN-like registry" and try to avoid "CRAN packages" everywhere as this is a "special" word in the community.

Reply:
I don't know about R, but in JS, a "npm package" is the preferred term, because npm is the defacto registry, but calling it "javascript package" is also fine.